File: Construct3/Construct3/Library/Telerik.OpenAccess.SDK/IDEIntegrations/templates/newWinFormGrid.vb
Public Partial Class %ClassName%
    Inherits System.Windows.Forms.Form


    Public Sub New()
        MyBase.New()

        'This call is required by the Windows Form Designer.
        InitializeComponent()

        'Add any initialization after the InitializeComponent() call

    End Sub


    Private Sub buttonSave_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles buttonSave.Click
        Me.ObjectProvider1.SaveAll()
    End Sub
    Private Sub buttonCancel_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles buttonCancel.Click
        Me.ObjectProvider1.CancelAll()
    End Sub
End Class

File: Construct3/Construct3/Library/Telerik.OpenAccess.SDK/IDEIntegrations/templates/newWinFormGrid.cs
//Copyright (c) %CompanyName%. All rights reserved.
//
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;

namespace %DefaultNamespace%
{
	/// <summary>
	/// Summary description for %ClassName%.
	/// </summary>
	public partial class %ClassName% : System.Windows.Forms.Form
	{
		public %ClassName%()
		{
			//
			// Required for Windows Form Designer support
			//
			InitializeComponent();

			//
			// Add any constructor code after InitializeComponent call
			//
			// Add the following code if you want to dispose on close
			this.Closed += new System.EventHandler(this.FormClosedEvent);
		}

		private void buttonSave_Click(object sender, System.EventArgs e)
		{
			this.objectProvider1.SaveAll();
		}

		private void buttonCancel_Click(object sender, System.EventArgs e)
		{
			this.objectProvider1.CancelAll();
		}

		private void FormClosedEvent(object sender, System.EventArgs e)
		{
			this.Dispose();
		}
	}
}
